packages/regl-worldview/src/commands/Triangles.js

```diff
@@ -109,11 +115,12 @@ const vertexColors = (regl) =>
       },
     },
 
-    blend,
-    depth: {
-      enable: true,
-      func: "<=",
-    },
+    blend: (context, props) => props.blend || defaultBlend,
+    depth: (context, props) =>
+      props.depth || {
+        enable: true,
+        func: "<=",
+      },
```

Comment:
nit: can we move the inline object into a `const` declared outside the function? Don't need to realloc this object every time we call this function if its unchanging.